Cargo.lock

```diff
@@ -1865,6 +1865,7 @@ dependencies = [
  "tempfile 3.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "untrusted 0.7.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "url 2.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
+ "walkdir 2.3.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
No idea what's happening here. I even went so far as to recreate the `Cargo.lock` file but it still wants to pull in 2.3.1?

Reply:
What if you use an equals requirement instead of a (default) caret requirement?  (We don't actually want to do that, but it'd be nice to understand.  This isn't a blocker.)

Reply:
I misunderstood caret requirements - "x.y.z" allows "x.*.*" higher versions, not just "x.y.*" higher versions, if the dependency is past 1.0.  I guess I'm used to 0.x dependencies :)